docs/topics/jvm/jvm-test-using-junit.md

```diff
@@ -0,0 +1,127 @@
+[//]: # (title: Test code using JUnit in JVM – tutorial)
+
+This tutorial shows you how to write a simple unit test and run it with the Gradle build tool.
+
+The example has the [kotlin.test](/api/latest/kotlin.test/index.html) library under the hood and runs the test using JUnit.
+
+To get started, first download and install the latest version of [IntelliJ IDEA](http://www.jetbrains.com/idea/download/index.html).
+
+## Add dependencies
+
+1. Open a Kotlin project in IntelliJ IDEA. If you don't have a project, [create one](/docs/tutorials/jvm-get-started.html#create-an-application).
+
+   > Specify **JUnit 4** as your test framework when creating a project.
+   >
+   {type="note"} 
+
+2. Open the `build.gradle(.kts)` file and add the following dependencies to the Gradle configuration to work with `kotlin.test` and `JUnit`:
+
+<tabs>
+
+   ```groovy
+   dependencies {
+       // Other dependencies.
+       // ...
+       testImplementation "org.jetbrains.kotlin:kotlin-test:1.3.11"
+       testImplementation "org.jetbrains.kotlin:kotlin-test-junit:1.3.11"
```

Comment:
Why are versions explicit here and not in the kotlin DSL example below?

Reply:
I've updated the sample to JUnit 5 and explicitly add versions to their dependencies (both Kotlin and Groovy)